internal/querycoord/task.go

```diff
@@ -1847,7 +1847,7 @@ func (ht *handoffTask) execute(ctx context.Context) error {
 				// we should copy a request because assignInternalTask will change DstNodeID of LoadSegmentRequest
 				clonedReq := proto.Clone(loadSegmentReq).(*querypb.LoadSegmentsRequest)
 				clonedReq.ReplicaID = replica.ReplicaID
-				tasks, err := assignInternalTask(ctx, ht, ht.meta, ht.cluster, []*querypb.LoadSegmentsRequest{clonedReq}, nil, true, nil, nil, replica.GetReplicaID(), ht.broker)
+				tasks, err := assignInternalTask(ctx, ht, ht.meta, ht.cluster, []*querypb.LoadSegmentsRequest{clonedReq}, nil, false, nil, nil, replica.GetReplicaID(), ht.broker)
```

Comment:
seems that wait is not used anymore, are we still need it?

Reply:
confirmed, not need this any more